tests/test_create_run.py

```diff
@@ -16,11 +16,11 @@ def teardown_module():
 
 def test_000_create_monitor():
     global monitor
-    monitor = cluster.create_monitor("/tmp/basic/monitor")
+    monitor = cluster.create_monitor("/tmp/create--run/monitor")
```

Comment:
maybe `create-run` or `create_run`

Reply:
Done using `create-run` ; I did like using `--run` in the name of the directory because we are testing `pg_autoctl create --run`, but yeah, that's a tad too much.